dashboard/src/modules/Dashboard/Templates.vue

```diff
@@ -44,31 +28,21 @@
       </v-card>
     </div>
 
-    <div
-      tabindex="-1"
-      class="v-stepper__step subtitle-2 v-stepper__step--active"
-    >
+    <div tabindex="-1" class="v-stepper__step subtitle-2 v-stepper__step--active">
       <span class="v-stepper__step__step">
         <v-icon color="primary"> mdi-mirror-rectangle </v-icon>
       </span>
       <div class="v-stepper__label font-weight-bold">
         CHOOSE A TEMPLATE
-        <small
-          >Select a template from the largest selection of free forms templates
-          available online</small
-        >
+        <small>Select a template from the largest selection of free forms templates
+          available online</small>
       </div>
     </div>
     <div>
       <v-list dense color="transparent">
         <v-list-item-group color="primary">
-          <v-list-item
-            v-for="item in temples"
-            :key="item.title"
-            dense
-            @dblclick="createFromTemplate(item)"
-            @click="previewTemplate(item)"
-          >
+          <v-list-item v-for="item in temples" :key="item.title" dense @dblclick="createFromTemplate(item)"
+            @click="previewTemplate(item)">
```

Comment:
keep it in multiple lines, is most easy to read the code

Reply:
done